ci/operations/set-cpu-entitlement-per-share.yaml

```diff
@@ -0,0 +1,3 @@
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=garden/properties/garden/experimental_cpu_entitlement_per_share_in_percent?
+  value: 0.195
```

Comment:
This is how the number got calculated: https://github.com/cloudfoundry/app-autoscaler-release/blob/fec20a167f5205dc82e1cdbb2fb50ff4bb1fab00/src/acceptance/app/dynamic_policy_test.go#L270-L272

Reply:
Hmm, can you put this or a link to the explanation maybe into the ops file as well as a comment. Just in case we come back and scratch our heads. :slightly_smiling_face: